config.mak.uname

```diff
@@ -652,7 +652,6 @@ ifeq ($(uname_S),MINGW)
 	USE_WIN32_IPC = YesPlease
 	USE_WIN32_MMAP = YesPlease
 	MMAP_PREVENTS_DELETE = UnfortunatelyYes
-	USE_NED_ALLOCATOR = YesPlease
```

Comment:
Hmm.

Imagine that 6 months from now, you read the commit message "Drop USE_NED_ALLOCATOR". And then see the diff that removes the line enabling `USE_NED_ALLOCATOR`. And now ask yourself:

- Would you know what this setting is about?
- Would you understand that this commit is modifying Git's own build configuration?
- Would you understand that this is in the _Windows-specific_ part of the build configuration?
- Would you understand why we want to drop it?
- Would you understand what the consequences are of dropping it?
- Would you interpret this patch as removing a _duplicate_ line, so that Git for Windows _still_ uses `nedmalloc`?

I would take a _hard_ guess that all of these questions would be answered with a resounding "No". And that's what the commit message has to fix.

I already pointed you to https://github.blog/2022-06-30-write-better-commits-build-better-projects/, but I do not see anything in this commit message that says "I've read it, and I applied the learnings".